processes/development/README.md

```diff
@@ -4,10 +4,14 @@
 
 These are the overarching principles that guide the actual details of the process:
 
-- **Quality is the most important thing. Above all else.**
-- Maximizing frequent output is more important than individual efficiency.
+- **Quality is the most important thing. Above all else.** This means taking time to do things correctly, and prioritizing bugs and errors above all else.
```

Comment:
I put it on the same line since i thought it was related. I could put it on the next line and indent it though. I don't really care :)